include/curl/curl.h

```diff
@@ -838,6 +838,10 @@ typedef enum {
    behavior is present. */
 #define CURLSSLOPT_REVOKE_BEST_EFFORT (1<<3)
 
+/* - REVOKE_BEST_EFFORT tells libcurl to use standard certificate store of
+   operating system. Currently implemented under MS-Windows. */
+#define CURLSSLOPT_NATIVE_CA (1<<4)
```

Comment:
Comment is wrong. Also I think you can get rid of the LIBCURL_WINDOWS_OPENSSL_NEVER_USE_WINDOWS_CA_STORE defines

Reply:
I just pushed it, thant